workers/task-driver/src/tasks/update_wallet.rs

```diff
@@ -263,45 +263,6 @@ impl Task for UpdateWalletTask {
 // -----------------------
 
 impl UpdateWalletTask {
-    /// Constructor
-    ///
-    /// Assumes that the state updates to the wallet have been applied in
-    /// the caller
-    #[allow(clippy::too_many_arguments)]
-    pub fn new(
```

Comment:
confirming we've removed the call sites for this throughout the codebase?

Reply:
we have not, that's part of the todo. It's more complicated than removing them -- they now need to go through the state module to queue tasks -- so more work to be done before the callsites change